packages/editor/src/store/selectors.js

```diff
@@ -659,7 +660,9 @@ export const getBlock = createSelector(
 	},
 	( state, clientId ) => [
 		state.editor.present.blocks.byClientId[ clientId ],
+		state.editor.present.blocks.attributesByClientId[ clientId ],
 		getBlockDependantsCacheBust( state, clientId ),
+		state.editor.present.blocks.order[ clientId ],
```

Comment:
> Unless I'm mistaken, since `getBlockDependantsCacheBust`calls `getBlock` for each child, we'll recursively depend on the entire set of IDs. Therefore, each `getBlock` only needs to worry about its own direct children, where it comes to `order`.

Isn't this already encompassed by including `getBlockDependantsCacheBust` as a dependant? i.e. this line shouldn't be necesary at all?

Reply:
Correct, it's redundant.